azurerm/resource_arm_api_management_api_schema.go

```diff
@@ -138,8 +139,15 @@ func resourceArmApiManagementApiSchemaRead(d *schema.ResourceData, meta interfac
 
 	if properties := resp.SchemaContractProperties; properties != nil {
 		d.Set("content_type", properties.ContentType)
-		if documentProperties := properties.SchemaDocumentProperties; documentProperties != nil {
-			d.Set("value", documentProperties.Value)
+		if documentProperties := properties.Document; documentProperties != nil {
+			// TODO: Make sure this makes sense
+			t := reflect.TypeOf(properties.Document)
+			value, ok := t.FieldByName("value")
+			if !ok {
+				log.Printf("[DEBUG] API Schema %q could not read value from schema", schemaID)
+			} else {
+				d.Set("value", value)
+			}
```

Comment:
⚠This has not been tested yet, and I'm not sure if it works as expected.
I don't have a proper test environment to test this in.

The `properties.Document` field is of type `interface{}`, but according to the REST API documentation, this should be a JSON document with the structure `{ "value": "..." }`.
There is hopefully a better way to deal with these cases in Go.